src/Components/GameCard/index.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import star1 from './cardsIcons/etoile1.svg';
+import star2 from './cardsIcons/etoile2.svg';
+import star3 from './cardsIcons/etoile3.svg';
+import star4 from './cardsIcons/etoile4.svg';
+import star5 from './cardsIcons/etoile5.svg';
+import './index.css';
+
+const GameCard = ({ heroApi, cardSize }) => {
+  const [basicInfo, setBasicInfo] = useState([]);
```

Comment:
If you can type later `basicInfo.name`, it means that basicInfo is an Object, not an array -> initial value === {}